src/ruby/puppet-server-lib/puppet/server/http_client.rb

```diff
@@ -50,8 +59,8 @@ def post(url, body, headers, options = {})
       end
 
       # http://en.wikipedia.org/wiki/Basic_access_authentication#Client_side
-      headers["Authorization"] =
-          "Basic #{Base64.strict_encode64 "#{credentials[:user]}:#{credentials[:password]}"}"
+      encoded = Base64.strict_encode64("#{credentials[:user]}:#{credentials[:password]}")
+      headers["Authorization"] = "Basic #{encoded}"
```

Comment:
This particular nesting of strings and hashes was breaking syntax highlighting. Also pretty difficult to read. I just pulled it out into a local variable.